extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -16,9 +16,14 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 	win.ads.runtime.disableBtf = false;
 	win.ads.runtime.unblockHighlyViewableSlots = false;
 
-	messageListener.register({dataKey: 'disableBtf', infinite: true}, function (msg) {
+	var btfQueue = [],
```

Comment:
Bleh I will have conflicts ;)

Reply:
Thanks for info, I need to hurry up then 😈